docs/src/main/sphinx/connector/query-table-function-ordering.fragment

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+    Some table functions may not preserve the order of the query result.
```

Comment:
The "Some table functions may not preserve the order of the query result" sentence is inaccurate. It's not the table function which does not preserve the order. It's the query engine.

Regardless, I think the sentence is irrelevant in this context. The fragment is included specifically within the docs of the query function within each connector, so, if anything, it should say "The query engine does not preserve the order of the results of this function". I would just remove it altogether, however.

Reply:
Removed.